image-layout.md

```diff
@@ -115,4 +116,42 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## Index 
+
+Index is intended to be generated from the content in an image-layout after any updates have been made.
```

Comment:
“any updates” → “any ref additions or removals”.  You shouldn't need to update it after other changes (e.g. blob addition or removal, or changing a ref's descriptor payload).